python/packages/ai/teams/app.py

```diff
@@ -251,8 +251,11 @@ def __selector__(context: TurnContext):
                 hits = re.match(select, text)
                 return hits is not None
 
-            i = context.activity.text.find(select)
```

Comment:
Hi @skoryky, we just had another fix for this via #1850- can you check if you are still getting this error with ME's?

Reply:
Awesome, that other PR takes care of it. Closing!